components/collection/drop/GenerativePreview.vue

```diff
@@ -60,10 +61,17 @@ const emit = defineEmits(['select'])
 const { accountId } = useAuth()
 const { chainProperties } = useChain()
 
+const STEP = 64
+const entropyRange = computed<[number, number]>(() => [
+  STEP * props.minted,
+  STEP * (props.minted + 1),
+])
+
 const getHash = (isDefault?: boolean) => {
   const ss58Format = isDefault
     ? chainProperties.value?.ss58Format
-    : getRandomInt(15000)
+    : getRandomIntFromRange(entropyRange.value[0], entropyRange.value[1])
```

Comment:
`chainProperties.value?.ss58Format` How about the default one? It should be also in the available range.

Reply:
![CleanShot 2024-01-13 at 08 17 50@2x](https://github.com/kodadot/nft-gallery/assets/44554284/64ffa0c2-d777-4c58-9e2e-aa441834ee91)

first entropyRange `[ 0, 64 ]` includes 0 and 2 which are the default ones for `ahp` and `ahk`

Reply:
I would use `entropyRange.value[0]` as the default value to avoid duplicate items minted by a user.

Reply:
but isn't it the same ?

ahp drop -> ss58Format is 0  and the first range is `[ 0, 64 ]` which has 0 , second range will not have 0 anymore, how do you get a duplicate by the same user ?

ahk same here it's 2 part of the first range after that you can't get 2 anymore 

I have no issue with making the change, but I don't see how you can get a duplicate with this code

Reply:
> ahp drop -> ss58Format is 0 and the first range is [ 0, 64 ] which has 0 , second range will not have 0 anymore, how do you get a duplicate by the same user ?

There is a case that a user could make the same drops.
If the user does not click the "Generative New" button, he will always get the drop with the default ss58Format value, which is zero. That means he could get the same drops as much as he wants. wdyt ?

Reply:
yep, absolulty 

updated the code https://github.com/kodadot/nft-gallery/pull/8967/commits/e37f3364bd7dda5b7bbd76b50fac0129750c9750